pkg/app/managedisc/factory.go

```diff
@@ -14,30 +12,26 @@ import (
 
 // Factory creates appdisc.Updater instances based on the app name.
```

Comment:
Comment needs to be updated as well.

Reply:
After changing from `managedisc` to `appdisc` and change `VisorManager` and `AppManager` to `*Updater` no need to change.